docs/docs/installation/windows.mdx

```diff
@@ -52,11 +52,12 @@ This installs a couple of things:
 
 For the `$PATH` to reload, a restart of your terminal is advised.
 
-:::warning Windows Defender
-To ensure Oh My Posh isn't blocked by Windows Defender, open an elevated PowerShell prompt and run the following command:
+:::warning Antivirus software
+Due to frequent updates of Oh My Posh, Antivirus software occasionally flags it (false positive). To ensure Oh My Posh isn't blocked you can either report it to your favorite Antivirus software as false positive (e.g. [Report a false positive/negative to Microsoft for analysis](https://docs.microsoft.com/en-us/microsoft-365/security/defender/m365d-autoir-report-false-positives-negatives#report-a-false-positivenegative-to-microsoft-for-analysis)) or create an exclusion for it.  
```

Comment:
Links need to be at the bottom, and you'll probably beed some newlines to solve the markdownlint remarks 😃

Reply:
Thanks for the fast response! I'll give my best 😉

Reply:
@JanDeDobbeleer Do you squash it or should I do a force push?

Reply:
Force push is fine!